### ISSUE TITLE
Hi！If the depth is too great, it will be wrong. What does this warning mean?

### ISSUE THREAD

**Issue by zaoyueri** [mkazhdan/PoissonRecon#247]
[WARNING] D:\posong3d\PoissonRecon-master\PoissonRecon-master\Src\FEMTree.IsoSurface.specialized.inl (Line 1886)
          IsoSurfaceExtractor<3,float,struct VectorTypeUnion<float,struct Point<float,3>,struct EmptyVectorType<float> > >::Extract
          bad average roots: 2

**Comment by mkazhdan** [comment]
Yep. That happens sometimes. It's an issue with the numerical solver which assumes a root in the range [0,1] but, because of numerical issues the root may be just outside. In that case the code clamps the root. 

The reconstruction should still be fine in this case.

**Comment by zaoyueri** [comment]
> 是的。这种情况有时会发生。这是数值求解器的问题，它假设根在 [0，1] 范围内，但由于数值问题，根可能就在外面。在这种情况下，代码会夹住根。
> 
> 在这种情况下，重建应该仍然没问题。

Yes, professor. I found that it was not caused by excessive depth. This reminder will be found when the depth is 6, 8, 12, which seems to be random. At the same time, the reconstruction effect will not be affected! Thanks for your reply, I have learned a lot.
